rust/threshold-schnorr/src/schnorr_example_rust/tests/tests.rs

```diff
@@ -110,12 +180,13 @@ fn test_impl(pic: &PocketIc, algorithm: SchnorrAlgorithm) {
                 signature_hex.clone(),
                 clone_and_reverse_chars(&message),
                 public_key_hex.clone(),
+                merkle_tree_root_hex.clone(),
                 algorithm,
             ))
             .unwrap(),
         );
 
-        assert!(!verification_reply.unwrap().is_signature_valid);
+        assert_ne!(verification_reply, successful_validation.clone());
```

Comment:
Here yes, changed. In 230-233 we expect an error but no a validation returning `false`. So this would need a `match`. And since this would be the only place where `assert_matches` would be used, I would prefer to leave it as is.